legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -116,7 +116,7 @@ def validate_roles(incorporation_json) -> Error:
         err_path = '/filing/incorporationApplication/parties/roles'
         msg.append({'error': 'Must have a maximum of one completing party', 'path': err_path})
 
-    if incorporation_json['filing']['incorporationApplication']['nameRequest']['legalType'] == 'BC':
+    if incorporation_json['filing']['business'].get('legalType', 'BC') == 'BC':
```

Comment:
Based on Linda's comment, this looks incorrect.
"My understanding is temporarily we are basing this on the button the person selects on the business registry page. Once names request is built, the legal type will be determined through Names Request"

Reply:
should the entire check be omitted then? It sounds like we can't guarantee the legalType is in the incorporation JSON currently

Reply:
@thorwolpert Yes, we can get entity type from the NR (but not currently as it is CR).

Wouldn't `filing.business.entityType` always be set from when the IA is created?

I'm not sure when that "button" will be implemented, but even so (and especially for numbered companies), that code will know what entity type to set, right?

Reply:
According to the req folks, it is set by the names flow.
For now, it could be left out until names gets working for us.
And for MVP we only handle this one legal type anyways, AND BC doesn't let us know it's NOT a Benefit Company. So it doesn't do anything we actually want / need.
+1 on just removing the check @peter-freshworks as you. suggested

Reply:
check has been removed with FUTURE tag